apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx

```diff
@@ -255,6 +287,19 @@ export const ApplicationCertificateWrapper: FunctionComponent<ApplicationWrapper
                                         updateCertType(values.get("certificateType") as CertificateTypeInterface);
                                     }
                                 }
+                                onBefore={
+                                    (event,value) => {
+                                        const certType  =  value as CertificateTypeInterface;
```

Comment:
Use ESLint auto-formatting.

```suggestion
                                        const certType  =  value as CertificateTypeInterface;
```